requirements.txt

```diff
@@ -0,0 +1 @@
+selenium (implicit requirement: Chrome)
```

Comment:
I've never actually set up a package with pip before, but the only requirement is "pip install selenium", which isn't much to ask someone to just type manually, in my opinion.

Using python3, as evidenced by the use of urllib as opposed to urllib2.